tests/object/tree/read.c

```diff
@@ -73,3 +73,59 @@ void test_object_tree_read__two(void)
 	git_object_free(obj);
 	git_tree_free(tree);
 }
+
+#define BIGFILE "bigfile"
+
+void test_object_tree_read__largefile(void)
+{
+	git_reference *ref;
+	git_commit *commit;
+	git_tree *tree;
+	git_oid oid;
+	const git_tree_entry *entry;
+	git_object *object;
+	git_buf file = GIT_BUF_INIT;
+	int fd;
+	fstore_t prealloc;
+	git_index *idx;
+
+	if (!cl_is_env_set("GITTEST_INVASIVE_FS_SIZE"))
+		cl_skip();
+
+	cl_git_pass(git_reference_lookup(&ref, g_repo, "refs/heads/master"));
+	cl_git_pass(git_repository_index(&idx, g_repo));
+
+	git_buf_puts(&file, git_repository_workdir(g_repo));
+	git_buf_joinpath(&file, file.ptr, BIGFILE);
+	cl_git_pass(git_buf_oom(&file));
```

Comment:
I'd rather avoid this call and wrap the previous to function calls in `cl_git_pass`